packages/relay-kit/src/packs/safe-4337/Safe4337Pack.ts

```diff
@@ -478,15 +478,17 @@ export class Safe4337Pack extends RelayKitBasePack<{
         preVerificationGas: BigInt(userOperation?.preVerificationGas || 0),
         maxFeePerGas: BigInt(userOperation?.maxFeePerGas || 0),
         maxPriorityFeePerGas: BigInt(userOperation?.maxPriorityFeePerGas || 0),
-        paymasterAndData: userOperation?.paymasterData || '0x',
-        signature: userOperation?.signature || '0x'
+        paymasterAndData: ethers.hexlify(
+          ethers.concat([userOperation?.paymaster || '0x', userOperation?.paymasterData || '0x'])
+        ),
+        signature: safeOperationResponse?.preparedSignature || '0x'
```

Comment:
`safeOperationResponse` is the function parameter, will always exist

Reply:
yep changed